kafka_utils/util/zookeeper.py

```diff
@@ -117,68 +117,123 @@ def get_topic_config(self, topic):
 
         :rtype : dict of configuration
         """
+        return self._get_entity_config(
+            "topics",
+            topic,
+            lambda topic: len(self.get_topics(topic_name=topic, fetch_partition_state=False)) > 0,
+        )
+
+    def set_topic_config(self, topic, value, kafka_version=(0, 10, )):
+        """Set configuration information for specified topic.
+
+        :topic : topic whose configuration needs to be changed
+        :value :  config value with which the topic needs to be
+            updated with. This would be of the form key=value.
+            Example 'cleanup.policy=compact'
+        :kafka_version :tuple kafka version the brokers are running on.
+            Defaults to (0, 10, x). Kafka version 9 and kafka 10
```

Comment:
Will this work on 1.1 as well?

Reply:
Ah good catch, I forgot to update the documentation, it works for kafka 0.9 and above (including 1.1, 2.X, etc...). I'll update it.